index.html

```diff
@@ -13,5 +13,52 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Acme Web Sulutions</title>
   </head>
-  <body></body>
+  <body>
+    <header class="site-header">
+      <div class="site-wrapper">
+        <div>header</div>
+      </div>
+    </header>
+    <section class="first-section">
+      <div class="site-wrapper">
+        <div class="first-section-container">
+          <div class="first-section-item">1</div>
+          <div class="first-section-item">2</div>
+        </div>
+      </div>
+    </section>
+    <section class="second-section">
+      <div class="site-wrapper">
+        <ul class="column-container">
+          <li class="column-item">1</li>
+          <li class="column-item">2</li>
+          <li class="column-item">3</li>
+        </ul>
+      </div>
+    </section>
+    <section class="third-section">1</section>
+    <div class="site-wrapper">
+      <section class="fourth-section">
+        <div class="fourth-section-item">
+          <div>1</div>
+        </div>
+      </section>
+    </div>
+    <div class="site-wrapper">
+      <section class="fifth-setion">
+        <div class="fifth-section-item">
+          <div>2</div>
+        </div>
+      </section>
+    </div>
+      <footer class="site-footer">
+        <div class="site-wraper">
+          <div class="footer-container">
+            <div class="">1</div>
+            <div class="">2</div>
+          </div>
+        </div>
+      </footer>
+    </div>
```

Comment:
This closing tag is extra. You can remove it.

Reply:
Done.